lms/djangoapps/support/static/support/jsx/program_enrollments/inspector.jsx

```diff
@@ -2,55 +2,174 @@ import React from 'react';
-        value={(props.learnerInfo && props.learnerInfo.user && props.learnerInfo.user.external_user_key) || ''}
-      />
+
+const renderUserSection = userObj => (
+  <div>
+    <h3>
+      edX account Info
+    </h3>
+    <div><span>Username: </span> {userObj.username}</div>
+    <div><span>Email: </span> {userObj.email}</div>
+    {userObj.external_user_key && (
+      <div><span>External User Key</span>: {userObj.external_user_key}</div>
+    )}
+    {userObj.sso ? (
+      <div>
+        <h4>Single Sign On Record:</h4>
+        <div><span>UID: </span>{userObj.sso.uid}</div>
+        <div><span>Identity Provider: </span>{userObj.sso.provider}</div>
+      </div>
+    ) : (
+      <div> There is no Single Signed On record associated with this user!</div>)}
+    <hr />
+  </div>
+);
+
+
+const renderVerificationSection = verificationStatus => (
+  <div>
+    <h3>
+      ID Verification
+    </h3>
+    <div><span>Status: </span> {verificationStatus.status}</div>
+    <div><span>Verification Error: </span> {verificationStatus.error}</div>
+    <div><span>Should Display Verification: </span> {verificationStatus.should_display}</div>
+    <div><span>Verification Expiration Date: </span>
+      {verificationStatus.verification_expiry}
     </div>
-    <Button label="Search" type="submit" className={['btn', 'btn-primary']} />
-  </form>
+    <hr />
+  </div>
+);
+
+const renderEnrollmentsSection = enrollments => (
+  <div>
+    <h3>
+      Program Enrollments
+    </h3>
+    {enrollments.map(enrollment => (
+      <div key={enrollment.program_uuid}>
+        <div>Program {enrollment.program_uuid} Enrollments</div>
+        <div> <span>Status: </span> {enrollment.status} </div>
+        <div> <span>Created: </span> {enrollment.created} </div>
+        <div> <span>Last updated: </span> {enrollment.modified} </div>
+        <div> <span>External User Key: </span> {enrollment.external_user_key} </div>
+        {enrollment.program_course_enrollments && enrollment.program_course_enrollments.map(
+          programCourseEnrollment => (
+            <div key={programCourseEnrollment.course_key}>
+              <br />
+              <h4>Course {programCourseEnrollment.course_key}</h4>
+              <div> <span>Status: </span> {programCourseEnrollment.status} </div>
+              <div> <span>Created: </span> {programCourseEnrollment.created} </div>
+              <div> <span>Last updated: </span> {programCourseEnrollment.modified} </div>
+              {programCourseEnrollment.course_enrollment && (
+                <div>
+                  <br />
+                  <h4>Linked course enrollment</h4>
+                  <div><span>Course ID: </span>
+                    {programCourseEnrollment.course_enrollment.course_id}
+                  </div>
+                  <div> <span>Is Active: </span>
+                    {programCourseEnrollment.course_enrollment.is_active}
```

Comment:
I noticed that the value for `Is Active` is missing in your screenshot.

![image](https://user-images.githubusercontent.com/11871801/77181700-17841600-6aa2-11ea-9da2-775496a178e9.png)

Note that React [ignores booleans, null, and undefined](https://reactjs.org/docs/jsx-in-depth.html#booleans-null-and-undefined-are-ignored) when rendering children, which is why the boolean doesn't show up here. Per the documentation, you should convert this boolean value into a string in order for it to show up.

```
String(programCourseEnrollment.course_enrollment.is_active)
```

You probably will want to do this anywhere you have an `undefined` as well. This would explain why I see missing values in the `ID Verification` section as well.